Update sk_SK app translation for 3.9.0

jose1711 opened the pull request:
Fixes #2700

ann0see reviewed (commented):
- [x] Translator listed in the `src/util.cpp`  https://github.com/jamulussoftware/jamulus/blob/fa204ad4f3025b7b2a1a8c1130ad998784faeb4a/src/util.cpp#L736
- [x] Punctuation and spacing consistent
- [x] Signal words consistent ("ASIO", "Buffer")
- [x] App translations: No untranslated strings (`grep unfinished -5 src/translation/translation_$TRANSLATION*.ts`) (there are some)
- [x] App translations: Only a single `.ts` file checked in (`.qm` in addition is also OK)

ann0see commented:
There are unfinished translations: Do you plan to translate them for this release?

```
$ grep unfinished -5 src/translation/translation_sk_SK.ts 
        <translation>Posuvný ovládač miestneho chvenia</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="115"/>
        <source>Server jitter buffer slider control</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="117"/>
        <source>Auto jitter buffer switch</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Sound Card Device</source>
        <translation type="vanished">Zariadenie zvukovej karty</translation>
    </message>
--
        <translation>Ak ovládač vyberiete počas aktívneho pripojenia, pripojenie je pozastavené, ovládač sa zmení a následne sa spojenie automaticky opäť obnoví.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="133"/>
        <source>Sound card device selector combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="142"/>
        <source>If you are using the kX ASIO driver, make sure to connect the ASIO inputs in the kX DSP settings panel.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="149"/>
        <source>Sound Card Channel Mapping</source>
        <translation>Mapovanie kanálov zvukovej karty</translation>
--
        <translation type="vanished"> vstupný/výstupný kanál (ľavý a pravý kanál) môžete vybrať iný kanál zvukovej karty.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="162"/>
        <source>Left input channel selection combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="164"/>
        <source>Right input channel selection combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="166"/>
        <source>Left output channel selection combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="168"/>
        <source>Right output channel selection combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="172"/>
        <source>Enable Small Network Buffers</source>
        <translation>Povoliť malú vyrovnávaciu pamäť pre sieť</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="179"/>
        <source>Enable small network buffers check box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="183"/>
        <source>Sound Card Buffer Delay</source>
        <translation>Oneskorenie vyrovnávacej pamäte zvukovej karty</translation>
--
        <translation type="vanished">Ak nastavenia oneskorenia vyrovnávacej pamäte nie je možné meniť, zvukový ovládač neumožňuje programu %1 meniť toto nastavenie. Vo Windows kliknite na tlačidlo Nastavenia zariadenia ASIO a otvorte panel s nastaveniami ovládača. Na Linuxe použite na zmenu veľkosti vyrovnávacej pamäte nástroj na konfiguráciu servera JACK.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="234"/>
        <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL lets you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="239"/>
        <source>Opens the driver settings. Note: %1 currently only supports devices with a sample rate of %2 Hz. You will not be able to select a driver/device which doesn&apos;t. For more help see jamulus.io.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="252"/>
        <source>128 samples setting radio button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="255"/>
        <source>256 samples setting radio button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="271"/>
        <source>Select the meter style to be used for the level meters. The Bar (narrow) and LEDs (round, small) options only apply to the mixerboard. When Bar (narrow) is selected, the input meters are set to Bar (wide). When LEDs (round, small) is selected, the input meters are set to LEDs (round, big). The remaining options apply to the mixerboard and input meters.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="280"/>
        <source>Language</source>
        <translation>Jazyk</translation>
--
        <translation>Vyberte jazyk, ktorý sa použije v používateľskom rozhraní.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="282"/>
        <source>Language combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="291"/>
        <source>and</source>
        <translation>a</translation>
--
        <translation>Zosilnenie vstupu</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="344"/>
        <source>This setting allows you to increase your input signal level by factors up to 10 (+20dB). If your sound is too quiet, first try to increase the level by getting closer to the microphone, adjusting your sound equipment or increasing levels in your operating system&apos;s input settings. Only if this fails, set a factor here. If your sound is too loud, sounds distorted and is clipping, this option will not help. Do not use it. The distortion will still be there. Instead, decrease your input level by getting farther away from your microphone, adjusting your sound equipment or by decreasing your operating system&apos;s input settings.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="357"/>
        <source>Input Boost combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="451"/>
        <source>Bar (narrow)</source>
        <translation>Paličky (úzke)</translation>
--
        <translation>Zvukové zariadenie</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="123"/>
        <source>Under the Windows operating system the ASIO driver (sound card) can be selected using %1. If the selected ASIO driver is not valid an error message is shown and the previous valid driver is selected. Under macOS the input and output hardware can be selected.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="187"/>
        <source>Three buffer sizes can be selected</source>
        <translation>Môžete si vybrať medzi tromi nastaveniami veľkosti vyrovnávacej pamäte</translation>
--
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="200"/>
        <location filename="../clientsettingsdlg.cpp" line="222"/>
        <source>Some sound card drivers do not allow the buffer delay to be changed from within %1. In this case the buffer delay setting is disabled and has to be changed using the sound card driver. Use the appropriate tool for the interface in use to adjust this buffer size. For example, if using ASIO, use the &quot;ASIO Device Settings&quot; button to open the driver settings panel or if using JACK, use a tool such as QjackCtl to adjust the buffer size. Other interfaces, such as Pipewire, would require their appropriate tool being used. Please refer to the interface manual.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="208"/>
        <source>If no buffer size is selected and all settings are disabled, this means a buffer size in use by the driver which does not match the values. %1 will still work with this setting but may have restricted performance.</source>
        <translation>Ak nie je vybraná žiadna veľkosť vyrovnávacej pamäte a všetky nastavenia sú neaktívne, znamená to, že veľkosť vyrovnávacej pamäte, ktorý používa ovládač nesedí s hodnotami. %1 bude s týmito nastavenia fungovať, ale výkon môže byť nižší.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="214"/>
        <source>The actual buffer delay has influence on the connection, the current upload rate and the overall delay. The lower the buffer size, the higher the probability of a red light in the status indicator (drop outs) and the higher the upload rate and the lower the overall delay.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="270"/>
        <source>Meter Style</source>
        <translation>Štýl ukazovateľa</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="277"/>
        <source>Meter Style combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="360"/>
        <source>Custom Directories</source>
        <translation>Vlastné adresáre</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="361"/>
        <source>If you need to add additional directories to the Connect dialog Directory drop down, you can enter the addresses here.&lt;br&gt;To remove a value, select it, delete the text in the input box, then move focus out of the control.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="368"/>
        <source>Custom Directories combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="371"/>
        <source>Audio Upstream Rate</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="372"/>
        <source>Depends on the current audio packet size and compression setting. Make sure that the upstream rate is not higher than your available internet upload speed (check this with a service such as speedtest.net).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="381"/>
        <source>Number of Mixer Panel Rows</source>
        <translation>Počet riadkov panela s mixérom</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="381"/>
        <source>Adjust the number of rows used to arrange the mixer panel.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="384"/>
        <source>Number of Mixer Panel Rows spin box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="386"/>
        <source>Feedback Protection</source>
        <translation>Ochrana pred spätnou väzbou</translation>
--
        <translation>Zapnite ochranu pred spätnou väzbou na detekciu akustickej väzby medzi mikrofónom a reproduktormi.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="391"/>
        <source>Feedback Protection check box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="394"/>
        <source>Audio Alerts</source>
        <translation>Zvukové upozornenia</translation>
--
        <translation>Povoľte zvukové upozornenie pri prijatí správy v chate a keď sa pripojí nový klient. Pre prehratie upozornenia možno budete potrebovať ďalšie zariadenie zvuku.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="399"/>
        <source>Audio Alerts check box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="445"/>
        <source>Fancy</source>
        <translation>Efektný</translation>
--
        <translation>Zvukové kanály</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="307"/>
        <source> mode will increase your stream&apos;s data rate. Make sure your upload rate does not exceed the available upload speed of your internet connection.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="318"/>
        <source>Audio channels combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="321"/>
        <source>Audio Quality</source>
        <translation>Kvalita zvuku</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="328"/>
        <source>Audio quality combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="331"/>
        <source>New Client Level</source>
        <translation>Úroveň pre nových klientov</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="340"/>
        <source>New client level edit box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="137"/>
        <source>If the ASIO4ALL driver is used, please note that this driver usually introduces approx. 10-30 ms of additional audio delay. Using a sound card with a native ASIO driver is therefore recommended.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="38"/>
        <location filename="../clientsettingsdlg.cpp" line="48"/>
        <location filename="../clientsettingsdlg.cpp" line="49"/>
--
        <translation>Miestny prelínač zvukového vstupu (ľavý/pravý)</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="150"/>
        <source>If the selected sound card device offers more than one input or output channel, the Input Channel Mapping and Output Channel Mapping settings are visible.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="184"/>
        <source>The buffer delay setting is a fundamental setting of %1. This setting has an influence on many connection properties.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="249"/>
        <source>64 samples setting radio button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="260"/>
        <source>ASIO Device Settings push button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Custom Directory Server Address</source>
        <translation type="vanished">Adresa vlastného adresárového servera</translation>
    </message>
--
        <translation>Stereo</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="59"/>
        <source>Controls the relative levels of the left and right local audio channels. For a mono signal it acts as a pan between the two channels. For example, if a microphone is connected to the right input channel and an instrument is connected to the left input channel which is much louder than the microphone, move the audio fader in a direction where the label above the fader shows %1, where %2 is the current attenuation indicator.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="76"/>
        <source>The jitter buffer compensates for network and sound card timing jitters. The size of the buffer therefore influences the quality of the audio stream (how many dropouts occur) and the overall delay (the longer the buffer, the higher the delay).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="81"/>
        <source>You can set the jitter buffer size manually for the local client and the remote server. For the local jitter buffer, dropouts in the audio stream are indicated by the light below the jitter buffer size faders. If the light turns to red, a buffer overrun/underrun has taken place and the audio stream is interrupted.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="90"/>
        <source>If the Auto setting is enabled, the jitter buffers of the local client and the remote server are set automatically based on measurements of the network and sound card timing jitter. If Auto is enabled, the jitter buffer size faders are disabled (they cannot be moved with the mouse).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="96"/>
        <source>If the Auto setting is enabled, the network buffers of the local client and the remote server are set to a conservative value to minimize the audio dropout probability. To tweak the audio delay/latency it is recommended to disable the Auto setting and to lower the jitter buffer size manually by using the sliders until your personal acceptable amount of dropouts is reached. The LED indicator will display the audio dropouts of the local jitter buffer with a red light.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>The buffer delay setting is a fundamental setting of this software. This setting has an influence on many connection properties.</source>
        <translation type="vanished">Nastavenie oneskorenia buffera je základným nastavením tohto programu. Toto nastavenie ovplyvňuje mnoho vlastností spojenia.</translation>
    </message>
--
        <translation>Vyberte vzhľad, ktorý sa aplikuje na hlavné okno programu.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="267"/>
        <source>Skin combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="286"/>
        <source>Selects the number of audio channels to be used for communication between client and server. There are three modes available:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="292"/>
        <source>These modes use one and two audio channels respectively.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="297"/>
        <source>Mono in/Stereo-out</source>
        <translation>Mono dnu/Stereo von</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="298"/>
        <source>The audio signal sent to the server is mono but the return signal is stereo. This is useful if the sound card has the instrument on one input channel and the microphone on the other. In that case the two input signals can be mixed to one mono channel but the server mix is heard in stereo.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="306"/>
        <source>Enabling </source>
        <translation>Zapnutím </translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="312"/>
        <source>In stereo streaming mode, no audio channel selection for the reverb effect will be available on the main window since the effect is applied to both channels in this case.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="322"/>
        <source>The higher the audio quality, the higher your audio stream&apos;s data rate. Make sure your upload rate does not exceed the available bandwidth of your internet connection.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="332"/>
        <source>This setting defines the fader level of a newly connected client in percent. If a new client connects to the current server, they will get the specified initial fader level if no other fader level from a previous connection of that client was already stored.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="403"/>
        <source>ASIO Device Settings</source>
        <translation>Nastavenia zariadenia ASIO</translation>
--
        <translation type="vanished">To, čo sem napíšete sa zobrazí pri vašom prelínači na mixéri po tom, ako sa pripojíte k Jamulus serveru. Táto menovka sa rovnako zobrazí každému klientovi, ktorý sa pripojí k tomu istému serveru ako vy.</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="519"/>
        <source>Alias or name edit box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="521"/>
        <source>Instrument picture button</source>
        <translation>Tlačidlo s obrázkom hud. nástroja</translation>
--
        <translation type="vanished">Tlačidlo s vlajkou krajiny</translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="525"/>
        <source>City edit box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="527"/>
        <source>Skill level combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../clientsettingsdlg.cpp" line="621"/>
        <source>Beginner</source>
        <translation>Začiatočník</translation>
--
        <translation>Zobrazí servery uvedené vo vybranom adresári. V pokročilých nastaveniach môžete pridať vlastné adresáre.</translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="48"/>
        <source>Directory combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="57"/>
        <source>Filters the server list by the given text. Note that the filter is case insensitive. A single # character will filter for those servers with at least one person connected.</source>
        <translation>Filtruje zoznam serverov podľa zadaného textu. Všimnite si, že filter neberie ohľad na veľkosť písmen. Samostatný znak # zobrazí iba servery, ku ktorému je pripojená aspoň jedna osoba.</translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="68"/>
        <source>Uncheck to collapse the server list to show just the server details. Check to show everyone on the servers.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="74"/>
        <source>Server List</source>
        <translation>Zoznam serverov</translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="75"/>
        <source>The Connection Setup window lists the available servers registered with the selected directory. Use the Directory dropdown to change the directory, find the server you want to join in the server list, click on it, and then click the Connect button to connect. Alternatively, double click on the server name to connect.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="80"/>
        <source>Permanent servers (those that have been listed for longer than 48 hours) are shown in bold.</source>
        <translation>Stále servery (tie, ktoré sú v zozname dlhšie ako 48 hodín) sú zobrazené tučným písmom.</translation>
--
        <translation>Adresa servera</translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="87"/>
        <source>If you know the server address, you can connect to it using the Server name/Address field. An optional port number can be added after the server address using a colon as a separator, e.g. %1. The field will also show a list of the most recently used server addresses.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="97"/>
        <source>Holds the current server address. It also stores old addresses in the combo box list.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>The Connection Setup window shows a list of available servers. Server operators can optionally list their servers by music genre. Use the List dropdown to select a genre, click on the server you want to join and press the Connect button to connect to it. Alternatively, double click on on the server name. Permanent servers (those that have been listed for longer than 48 hours) are shown in bold.</source>
        <translation type="vanished">Okno Nastavenie pripojenia zobrazuje zoznam dostupných serverov. Prevádzkovatelia serverov môžu voliteľne zadať hudobný žáner. V rozbaľovacej ponuke vyberte žáner, kliknite na server, ku ktorému sa chcete pripojiť a kliknutím na tlačidlo Pripojiť sa pripojíte k serveru. Prípadne stačí dvakrát kliknúť na meno servera. Trvalé servery (také, ktoré sú v zozname viac ako 48 hodín) sú zobrazené tučným písmom.</translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="96"/>
        <source>Server address edit box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Server List Selection</source>
        <translation type="vanished">Výber zoznamu serverov</translation>
    </message>
--
        <translation type="vanished">Zoznam serverov bude filtrovaný na základe zadaného textu. Všimnite si, že filter neberie ohľad na veľkosť písmen.</translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="59"/>
        <source>Filter edit box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../connectdlg.cpp" line="67"/>
        <source>Show All Musicians</source>
        <translation>Zobraziť všetkých hudobníkov</translation>
--
        <translation>Zoznam pripojených klientov</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="176"/>
        <source>Click the button to open the dialog that allows the main recording directory to be selected.  The chosen value must exist and be writeable (allow creation of sub-directories by the user %1 is running as).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="183"/>
        <source>The current value of the main recording directory. The chosen value must exist and be writeable (allow creation of sub-directories by the user %1 is running as). Click the button to open the dialog that allows the main recording directory to be selected.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="195"/>
        <source>Custom Directory address</source>
        <translation>Adresa vlastného adresára</translation>
--
        <translation>Adresa vlastného adresára je adresa, na ktorej sa nachádza adresár so zoznamom serverov, do ktorého sa má tento server pridať.</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="204"/>
        <source>Server List Filename dialog push button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="205"/>
        <location filename="../serverdlg.cpp" line="213"/>
        <source>Server List Filename</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="206"/>
        <source>Click the button to open the dialog that allows the server list persistence file name to be set. The user %1 is running as needs to be able to create the file name specified although it may already exist (it will get overwritten on save).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="212"/>
        <source>Server List Filename text box (read-only)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="214"/>
        <source>The current value of server list persistence file name. The user %1 is running as needs to be able to create the file name specified although it may already exist (it will get overwritten on save). Click the button to open the dialog that allows the server list persistence file name to be set.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="221"/>
        <source>Clear the server list file name button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="222"/>
        <source>Clear Server List Filename</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="223"/>
        <source>Click the button to clear the currently selected server list persistence file name. This will prevent persisting the server list until a new value is selected.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="227"/>
        <source>Start Minimized on Operating System Start</source>
        <translation>Spustiť minimalizované pri štarte operačného systému</translation>
--
        <translation>Názov servera</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="94"/>
        <source>Server name line edit</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="97"/>
        <source>Location City</source>
        <translation>Umiestnenie/mesto</translation>
--
        <translation>Tu môžete nastaviť, v ktorom meste sa server nachádza. Ak je mesto zadané, zobrazí sa klientom v okne so zoznamom serverov.</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="105"/>
        <source>City where the server is located line edit</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Location country</source>
        <translation type="vanished">Umiestnenie/krajina</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="174"/>
        <source>Display dialog to select recording directory button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="175"/>
        <location filename="../serverdlg.cpp" line="182"/>
        <source>Main Recording Directory</source>
        <translation>Hlavný adresár na nahrávanie</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="181"/>
        <source>Main recording directory text box (read-only)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="189"/>
        <source>Clear the recording directory button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="190"/>
        <source>Clear Recording Directory</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="191"/>
        <source>Click the button to clear the currently selected recording directory. This will prevent recording until a new value is selected.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="119"/>
        <source>Checkbox to turn on or off server recording</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="120"/>
        <source>Enable Recorder</source>
        <translation>Povoliť nahrávanie</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="121"/>
        <source>Checked when the recorder is enabled, otherwise unchecked. The recorder will run when a session is in progress, if (set up correctly and) enabled.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="148"/>
        <source>Current session directory text box (read-only)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="152"/>
        <source>Current Session Directory</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="153"/>
        <source>Enabled during recording and holds the current recording session directory. Disabled after recording or when the recorder is not enabled.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="130"/>
        <source>Recorder status label</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="56"/>
        <source>Directory Type combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="60"/>
        <source>Directory</source>
        <translation>Adresár</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="61"/>
        <source>Select &apos;%1&apos; not to register your server with a directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="62"/>
        <source>Select one of the genres to register with that directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="63"/>
        <source>Or select &apos;%1&apos; and specify a Custom Directory address on the Options tab to register with a custom directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="67"/>
        <source>For any value except &apos;%1&apos;, this server registers with a directory so that a %2 user can select this server in the client connect dialog server list when they choose that directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="73"/>
        <source>The registration of the server is renewed periodically to make sure that all servers in the connect dialog server list are actually available.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="81"/>
        <source>When a value other than &quot;%1&quot; is chosen for Directory, this will show whether registration is successful. If the registration failed, please choose a different directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="132"/>
        <source>Recorder Status</source>
        <translation>Stav nahrávania</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="134"/>
        <source>No recording directory has been set or the value is not useable. Check the value in the Options tab.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="138"/>
        <source>Recording has been switched off by the UI checkbox.</source>
        <translation>Nahrávanie bolo vypnuté pomocou začiarkavacieho políčka.</translation>
--
        <translation>Nahrávanie bolo vypnuté, buď prostredníctvom začiarkavacieho políčka alebo bol prijatý signál SIGUSR2.</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="145"/>
        <source>If the recording directory is not useable, the problem will be displayed in place of the session directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="159"/>
        <source>Server welcome message edit box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="160"/>
        <source>Server Welcome Message</source>
        <translation>Uvítacia správa servera</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="161"/>
        <source>A server welcome message text is displayed in the chat window if a musician enters the server. If no message is set, the server welcome is disabled.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="167"/>
        <source>Language</source>
        <translation>Jazyk</translation>
--
        <translation>Vyberte jazyk, ktorý sa použije v rozhraní aplikácie.</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="171"/>
        <source>Language combo box</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="737"/>
        <source>Now a directory</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <source>Custom Directory</source>
        <translation type="vanished">Vlastný adresár</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="201"/>
        <source>Custom Directory line edit</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="245"/>
        <source>&amp;Hide %1 server</source>
        <translation>Sk&amp;ryť %1 server</translation>
--
        <translation>CHYBA</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="125"/>
        <source>Request new recording button</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="126"/>
        <source>New Recording</source>
        <translation>Nová nahrávka</translation>
--
        <translation type="vanished"> Server</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="228"/>
        <source>If the start minimized on operating system start check box is checked, the server will be started when the operating system starts up and is automatically minimized to a system task bar icon.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="88"/>
        <source>The server name identifies your server in the connect dialog server list at the clients.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="108"/>
        <source>Country/Region</source>
        <translation>Krajina/Región</translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="109"/>
        <source>Set the country or region where the server is running. Clients will show this location in their connect dialog&apos;s server list.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="116"/>
        <source>Combo box for location of this server</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="132"/>
        <source>Displays the current status of the recorder.  The following values are possible:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="142"/>
        <source>There is no one connected to the server to record.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="143"/>
        <source>The performers are being recorded to the specified session directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../serverdlg.cpp" line="144"/>
        <source>NOTE</source>
        <translation>POZNÁMKA</translation>
--
        <translation>Nebol nájdený zvukový ovládač ASIO.</translation>
    </message>
    <message>
        <location filename="../sound/asio/sound.cpp" line="555"/>
        <source>Please install an ASIO driver before running %1. If you own a device with ASIO support, install its official ASIO driver. If not, you&apos;ll need to install a universal driver like ASIO4ALL.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sound/jack/sound.cpp" line="47"/>
        <source>JACK couldn&apos;t be started automatically. Please start JACK manually and check for error messages.</source>
        <translation>Server JACK sa nepodarilo spustiť automaticky. Prosím, spustite JACK server ručne a skontrolujte chybové hlásenia.</translation>
--
        <translation>Vybrané zvukové zariadenie pre vstup už nie je v systéme dostupné. Prosím, skontrolujte, či je vaše zariadenie správne pripojené.</translation>
    </message>
    <message>
        <location filename="../sound/coreaudio-mac/sound.cpp" line="394"/>
        <source>The sample rate on the current input device isn&apos;t %1 Hz and is therefore incompatible. Please select another device or try setting the sample rate to %1 Hz manually via Audio-MIDI-Setup (in Applications-&gt;Utilities).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sound/coreaudio-mac/sound.cpp" line="406"/>
        <source>The audio output device is no longer available. Please check if your output device is connected correctly.</source>
        <translation>Vybrané zvukové zariadenie pre výstup už nie je dostupné. Prosím, skontrolujte, či je vaše zariadenie správne pripojené.</translation>
    </message>
    <message>
        <location filename="../sound/coreaudio-mac/sound.cpp" line="415"/>
        <source>The sample rate on the current output device isn&apos;t %1 Hz and is therefore incompatible. Please select another device or try setting the sample rate to %1 Hz manually via Audio-MIDI-Setup (in Applications-&gt;Utilities).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sound/coreaudio-mac/sound.cpp" line="463"/>
        <source>The stream format on the current input device isn&apos;t compatible with this software. Please select another device.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sound/coreaudio-mac/sound.cpp" line="474"/>
        <source>The stream format on the current output device isn&apos;t compatible with %1. Please select another device.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sound/coreaudio-mac/sound.cpp" line="727"/>
        <source>The buffer sizes of the current input and output audio device can&apos;t be set to a common value. Please select different input/output devices in your system settings.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>CSoundBase</name>
    <message>
```

ann0see commented:
You should rebase this Pull request as described here to fix the conflicts: https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md#updating-a-prfixing-conflicts

jose1711 commented:
> There are unfinished translations: Do you plan to translate them for this release?

No, no further plans.

jose1711 commented:
> You should rebase this Pull request as described here to fix the conflicts: https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md#updating-a-prfixing-conflicts Best would be to not commit the qm file

Removed `.qm` from the commit, rebased and force-pushed.

ann0see commented:
Please don’t remove the qm file. (Doing so breaks the build). Since you rebased the PR you should be able to re generate it.

jose1711 commented:
> Please don’t remove the qm file. (Doing so breaks the build). Since you rebased the PR you should be able to re generate it.

Done.

ann0see reviewed (approved):
Ok. Thanks. Approving since Checklist is ok and CI green.

pljones reviewed (approved):
I can't read Polish at all but approving.
